packages/functions/src/error.ts

```diff
@@ -51,6 +51,10 @@ const errorCodeMap: { [name: string]: FunctionsErrorCode } = {
 /**
  * An explicit error that can be thrown from a handler to send an error to the
```

Comment:
Can we put the old interface doc text here? I think it's less confusing for public consumption. It was "An error returned by the Firebase Functions client SDK." The link to the FunctionsErrorCode part is fine, that's good to add, it's just the old text on the class comment seems more internal-facing.